bokeh/core/compat/__init__.py

```diff
@@ -24,3 +24,9 @@
 .. _mplexporter: https://github.com/mpld3/mplexporter/tree/master/mplexporter
 
 '''
+from ...util.deprecation import deprecated
+
+deprecated(
+    "MPL compatibility can no longer be successfully maintained, and is now deprecated. " +
+    "All MPL compat functions will be removed completely on the release of Bokeh 1.0. " +
```

Comment:
Well, except that I'd remove examples *immediately*, as soon as anything becomes deprecated, to reduce the number of people who start relying on anything scheduled to disappear...